pkg/inventory/inventory-client_test.go

```diff
@@ -156,27 +157,29 @@ func TestMerge(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			// Create the local inventory object storing "tc.localObjs"
-			invClient, _ := NewInventoryClient(tf)
-			invClient.SetDryRun(true)
-			// Create a fake builder to return "tc.clusterObjs" from
-			// the cluster inventory object.
-			fakeBuilder := FakeBuilder{}
-			fakeBuilder.SetInventoryObjs(tc.clusterObjs)
-			invClient.builderFunc = fakeBuilder.GetBuilder()
-			// Call "Merge" to create the union of clusterObjs and localObjs.
-			pruneObjs, err := invClient.Merge(tc.localInv, tc.localObjs)
-			if tc.isError {
-				if err == nil {
-					t.Fatalf("expected error but received none")
+			for _, drs := range common.Strategies {
```

Comment:
For all the tests where we run with multiple configurations, move the for loop outside of the `t.Run` call. That way each configuration will be considered a separate test and it makes it easier to debug when a test fails.

Reply:
I get the lint error when I try to do that

`Using the variable on range scope `drs` in function literal`

Reply:
Just change the for loop to get around that:
```
for i := range common.Strategies {
  drs := common.Strategies[i]
  t.Run(...
}
```

Reply:
That worked. Thanks.